google_cloud_run_deploy/templates/terraform_default.tf

```diff
@@ -118,6 +124,10 @@ resource "google_cloud_run_service" "run_service" {
           name  = "BENTOML_PORT"
           value = var.port
         }
+        env {
+          name  = "BENTOML_CONFIG_OPTIONS"
+          value = var.bentoml_config_options
+        }
```

Comment:
I think what might make more sense is to add the option to add any arbitrary environment variables in the bentoctl deployment.yaml. Each operator will read and set the env vars from the deployment.yaml so we don't have to add this individually. @jjmachan, thoughts?

Reply:
To unblock you in the short term, @judahrand. I think you can fork this repo and add it as a custom operator.

Reply:
@ssheng yes that is a cleaner way to add environment vars. I'll add something from bentoctl's side

Reply:
Sounds good, I'll also have a quick look at this today.